test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -450,19 +449,18 @@ def random_name(self, length=16):
         return str().join(random.choice(string.ascii_lowercase)
                           for _ in range(length))
 
-    def load_hook(self, url):
+    def load_hook(self, filename):
         """
         Import and enable a hook pointed to by the URL specified.
         """
         try:
-            conn = urllib2.urlopen(url)
-            script = conn.read()
-            conn.close()
+            with open(filename, 'r') as fd:
+                script = fd.read()
         except:
             script = ''
-        self.assertTrue(script != '')
-        hook_file = os.path.join(os.sep, 'tmp', self.random_name())
-        with open(hook_file, "w") as fd:
+        self.assertNotEqual(script, '')
```

Comment:
Why don't you put this except in the except block?  It's when script gets set to ''

Reply:
Good point.